### PR TITLE
Print merged Schema with directives.

### PR DESCRIPTION
**Pull request by hsehweil**
[Summary]
This is to fix the issue where directives where not being generated after merging schema files.

*Issue #, if available:*

*Description of changes:*
add the method as a workaround for the issue of not being able to print the
 directives fields for the merged schema when using printSchema method

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Options:
   -s, --schema [pattern]      Use a glob path that would define all of your schema files to merge them into a valid schema. (default: "").
   -r, --rules [pattern]       The file path for your custom rules to validate your operations, and your merged schema. To learn abot how to write your custom rules: check the README.md file  (default: "").
   -p, --operations [pattern]  Use a glob that that contains your graphql operation files to test against the merged schema file. (default: "").
+  -d, --disableDirectives [pattern] By default will merge the directives, unless this was set to true.
```
Comment (brockgibson) [inline README.md]: Is the pattern part applicable?

File: src/cli-launcher.ts [hunk 2]
```diff
@@ -11,27 +11,35 @@ import * as graphql from 'graphql';
...
   .option('-s, --schema [pattern]',
-   'Use a glob path that would define all of your schema files to merge them into a valid schema.', '')
+    'Use a glob path that would define all of your schema files to merge them into a valid schema.', '')
   .option('-r, --rules [pattern]',
-   'The file path for your custom rules to validate your operations, and your merged schema.', '')
+    'The file path for your custom rules to validate your operations, and your merged schema.', '')
   .option('-p, --operations [pattern]',
-   'Use a glob that that contains your graphql operation files to test against the merged schema file.', '')
+    'Use a glob that that contains your graphql operation files to test against the merged schema file.', '')
+  .option('-d, --disableDirectives [pattern]',
```
Comment (brockgibson) [inline src/cli-launcher.ts]: Same thing about pattern.

File: README.md (hunk 1 above)
Comment (hsehweil) [reply README.md]: done.

File: src/cli-launcher.ts (hunk 2 above)
Comment (hsehweil) [reply src/cli-launcher.ts]: done.

**Comment by hsehweil** [comment]
done, ship it.

**Pull request merged**